apps/wingcli-v2/turbo.json

```
@@ -0,0 +1,14 @@
+{
+    "$schema": "https://turborepo.org/schema.json",
+    "extends": ["//"],
+    "pipeline": {
+      "compile": {
+        "dependsOn": ["@winglang/wingii#compile", "@winglang/wingc#compile"],
+        "outputs": []
+      },
+      "test": {
+        "dependsOn": ["compile", "@winglang/sdk#compile"]
+      }
```

None of the options are needed for compile (since wingii and wingc are deps in the package.json, they'll automatically be included). `test` also technically doesn't need to depend on `compile`

```suggestion
      "compile": {},
      "test": {
        "dependsOn": ["@winglang/sdk#compile"]
      }
```

---

TIL - thanks